### DISCUSSION TITLE
How to make bootstrap notification/ toast to appear on canvas when it's on full screen

### DISCUSSION THREAD

**Discussion by ChanduRach** [GrapesJS/grapesjs#4811] [Q&A]
I want to display a notification when the canvas is set to full screen

Problem:  it's hidden behind the canvas/iframe

![image](https://user-images.githubusercontent.com/25901390/209671555-c8589fb6-8c90-4e6e-ac6e-120c76876464.png)

**Comment by artf** [comment]
I guess it's might be only a z-index related issue.

**Reply by ChanduRach** [reply]
No, i tried but its not working
